README.md

```diff
@@ -72,9 +72,11 @@ Example:
 
 ![Configuration](assets/configuration.png)
 
+To get, `ACROLINX_GENERIC_TOKEN` refer Acrolinx [documentation](https://support.acrolinx.com/hc/en-us/articles/10306041280274) on Acrolinx single sign-on.
```

Comment:
Do you think this link is sufficient? I'm not able to link the generic token & sso :( 

There is [section](https://support.acrolinx.com/hc/en-us/articles/10306041280274#UUID-3df00e54-2c7b-cfd9-9408-5ca9c0c10318_id_SetUpAcrolinxforSingleSignon-WhichSingleSign-OnMethodsDoesAcrolinxSupport) on the page but its tough to judge how to get or whom to ask for generic token.

Reply:
@pcdeshmukh Updated. See https://github.com/acrolinx/Acrolinx-Installer-for-AEMaaCS/pull/4/commits/fc983bffa06aa4a901eaa44256087bd1022ef7a8